src/main/java/com/github/jenkinsci/zaproxyplugin/ZAPDriver.java

```diff
@@ -1265,10 +1266,9 @@ private String setUpUser(BuildListener listener, ClientApi zapClientAPI, String
         // means that any value we add for the configuration values has to be
         // URL encoded.
         StringBuilder userAuthConfig = new StringBuilder();
-        userAuthConfig.append("username=").append(URLEncoder.encode(username, "UTF-8"));
-        userAuthConfig.append("&password=").append(URLEncoder.encode(password, "UTF-8"));
+        if ( authMethod.equals(SCRIPT_BASED)) { userAuthConfig.append("Username=").append(URLEncoder.encode(username, "UTF-8")); userAuthConfig.append("&Password=").append(URLEncoder.encode(password, "UTF-8")); }
```

Comment:
I see ! So basically the API receives them and propagates the params to the ZEST script where the problem happen, anyway i will verify the source of the problem before reporting to the relevant teams. Meanwhile I will check the extension at least as precautionary measure.